pkg/aututils/aututils.go

```diff
@@ -100,14 +100,14 @@ func PoolRuleByAvailableCapacity(usage, scalePercentage uint64, expandType strin
 func PVCRuleByTotalSize(capacity int, scalePercentage int, maxSize string) apapi.AutopilotRule {
 	apRuleObject := apapi.AutopilotRule{
 		ObjectMeta: meta_v1.ObjectMeta{
-			Name: fmt.Sprintf("pvc-total-size-%s", strings.ToLower(maxSize)),
+			Name: fmt.Sprintf("pool-total-%d-scale-%d", capacity, scalePercentage),
 		},
 		Spec: apapi.AutopilotRuleSpec{
 			Conditions: apapi.RuleConditions{
 				Expressions: []*apapi.LabelSelectorRequirement{
 					{
 						Key:      PxVolumeTotalCapacityMetric,
-						Operator: apapi.LabelSelectorOpGt,
+						Operator: apapi.LabelSelectorOpLt,
```

Comment:
why Lt ?

Reply:
you want to resize PVC until the total size is less than provided value, otherwise it will constantly resize until maxsize if provided. We have the same logic in PoolRuleByTotalSize.